app/js/features/network/selectors.js

```diff
@@ -1,4 +1,14 @@
 export const isReady = state => state.network.ready;
+export const error = state => state.network.error;
+export const errorTip = state => {
```

Comment:
Too late because of merged, but you bind component and logic too much here probably
I would have named that: `noProviderError: boolean`.

Also because that way i18n won't work well :)